src/cordova/build.action.mjs

```diff
@@ -30,6 +30,13 @@ export async function main(...parameters) {
 
   await runAction('cordova/setup', ...parameters);
 
+  if (platform === 'osx' && buildMode === 'release') {
+    console.warn(
+      'Production MacOS builds currently must be done in XCode due to a cordova issue. Please open platforms/osx/Outline.xcodeproj to continue.'
+    );
+    return;
```

Comment:
build currently doesn't work at all for macos so it would have the same effect, and if i just let it throw we get a huge error which would bury this helper text

Reply:
We should make the command fail. Otherwise the caller will see a successful run, and that's problematic, as we saw in our CI where it was broken, but succeeding.

Reply:
They won't see a successful run - building release via the command line currently doesn't work at all and will throw an error - but I suppose an error we control is clearer than whatever it is cordova dumps out, so I made this throw regardless.